softgroup/services/authorization/authorization-impl/src/main/java/handler/AuthorizationRequestLoginHandler.java

```diff
@@ -0,0 +1,31 @@
+package handler;
+
+import com.softgroup.authorization.api.message.LoginRequest;
+import com.softgroup.authorization.api.message.LoginResponse;
+import com.softgroup.authorization.api.router.AuthorizationRequestHandler;
+import com.softgroup.authorization.api.router.AuthorizationRouterHandler;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import com.softgroup.common.router.api.AbstractRequestHandler;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by polev on 25.02.2017.
+ */
+
+@Component
+public class AuthorizationRequestLoginHandler
```

Comment:
@MariiaPolova No need to put Authorization to handler name. It's already in authorization module.
Better rename this to LoginRequestHandler